tests/tt_metal/tt_metal/unit_tests_frequent/module.mk

```diff
@@ -0,0 +1,31 @@
+# Every variable in subdir must be prefixed with subdir (emulating a namespace)
+
+TT_METAL_UNIT_TESTS_FREQUENT_SRCS_HOME = tests/tt_metal/tt_metal/unit_tests_frequent
+
+TT_METAL_UNIT_TESTS_FREQUENT = ${TT_METAL_UNIT_TESTS_FREQUENT_SRCS_HOME}/tests_main.cpp
+TT_METAL_UNIT_TESTS_FREQUENT += $(wildcard ${TT_METAL_UNIT_TESTS_FREQUENT_SRCS_HOME}/*/*.cpp)
+TT_METAL_UNIT_TESTS_FREQUENT += $(wildcard ${TT_METAL_UNIT_TESTS_FREQUENT_SRCS_HOME}/*/*/*.cpp)
```

Comment:
Hmm... I'm wondering if you can just replace these 3 lines with:

`TT_METAL_UNIT_TESTS_FREQUENT += $(wildcard ${TT_METAL_UNIT_TESTS_FREQUENT_SRCS_HOME}/**/*.cpp)`

Could we give it a try?

Reply:
Seems like it works, updated in latest rebase